WebDevStudios/wd_s
# Bumps copy-webpack-plugin to 6.4.1

Closes #579

### DESCRIPTION
We can't bump to v7 due to a breaking change, but this updates to the latest release version of 6.4.1.

### STEPS TO VERIFY
1. Check out the branch
2. Run `npm i`
3. Run `npm run start` and test some changes
4. Run `npm run build`

Overall, just test to make sure all of the build functions still work without issue.
